ui/imports/shared/popups/SendModal.qml

```diff
@@ -155,22 +156,30 @@ StatusDialog {
         onSelectedIndexChanged: store.switchSenderAccount(selectedIndex)
     }
 
-    StackLayout {
-        id: stack
+
+    ColumnLayout {
+        id: group1
+
         anchors.fill: parent
-        currentIndex: 0
-        clip: true
-        ColumnLayout {
-            id: group1
+
+        // Workaround for https://bugreports.qt.io/browse/QTBUG-87804
```

Comment:
Nice finding. Some time ago I could not figure out a popup layout issue and wondering if this might be the reason.